### PR TITLE
Adds the new replace flow to the video block

### PR DESCRIPTION
**Pull request by draganescu**
## Description
There is a new replace flow offered by a component which is currently implemented in the Image Block. This PR implements the new flow in the Audio Block.

## How has this been tested?
Tested locally.

## Screenshots 

Screenshot 2019-12-16 at 13 27 34

### CODE REVIEW

**Review by Soean — CHANGES_REQUESTED** [review]
The replacement flow works great 👍. If we don't use the state `editing` anymore, we can remove it from this component.

**Comment by draganescu** [comment]
Hi @Soean I removed the useless state in [960fdaa](https://github.com/WordPress/gutenberg/pull/19162/commits/960fdaaeacc7d52fcbfccaef87c53ac6b9b3d9b8). Thank you for reviewing all these!

File: packages/block-library/src/video/edit.js [hunk 1]
```diff
@@ -47,10 +47,6 @@ class VideoEdit extends Component {
 		super( ...arguments );
 		// edit component has its own src in the state so it can be edited
```
Comment (Soean) [inline packages/block-library/src/video/edit.js]: The `src` state was removed in #6820, so we can remove this comment.

File: packages/block-library/src/video/edit.js [hunk 2]
```diff
@@ -95,8 +90,7 @@ class VideoEdit extends Component {
 		const { attributes, setAttributes } = this.props;
 		const { src } = attributes;
 
-		// Set the block's src from the edit component's state, and switch off
-		// the editing UI.
+		// Set the block's src from the edit component's state
```
Comment (Soean) [inline packages/block-library/src/video/edit.js]: We also can remove this comment, it's not valid since #6820

**Review by Soean — APPROVED** [review]
Works great.
We can remove some old comments. Maybe we can do it in another PR.

**Review by Soean — APPROVED** [review]
Works great, so we can merge it.

**Pull request merged**